lib/src/components/Textarea/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { forwardRef } from 'react'
+
+import { classNames } from '@/utils'
+
+import textareaStyles from './textarea.module.scss'
+import type { TextareaProps } from './types'
+
+const cx = classNames(textareaStyles)
+
+export const Textarea = forwardRef<HTMLTextAreaElement, TextareaProps>(
+  ({ minRows = 5, variant = 'default', ...rest }, ref) => (
+    <textarea
+      className={cx('root', variant && `variant-${variant}`)}
```

Comment:
as variant is always defined, you could replace with `className={cx('root', `variant-${variant}`)}`